examples/rmg/commented/input.py

```diff
@@ -150,6 +150,16 @@
 )
 
 options(
+    	#provides a name for the seed mechanism produced at the end of an rmg run default is 'Seed'
+    name='RMGButane',	
```

Comment:
Yeah, it should probably be easier to distinguish from the real libraries, I think I'll just name it 'SeedName'.  

I know, but I think use of these as libraries is highly unlikely (since the rates are either from other libraries or from rate rules) so I still think Seed makes the most sense for the user.